vassal-app/src/main/java/VASSAL/build/module/map/DrawPile.java

```diff
@@ -279,7 +290,11 @@ public boolean isDisplayNames() {
       EXPRESSIONCOUNTING,
       COUNTEXPRESSIONS,
       RESTRICT_OPTION,
-      RESTRICT_EXPRESSION
+      RESTRICT_EXPRESSION,
+      DRAW_MULTIPLE_MESSAGE,
+      DRAW_SPECIFIC_MESSAGE,
+      FACE_UP_MESSAGE,
+      FACE_DOWN_MESSAGE
```

Comment:
Re-order these (and getDescriptions() and getTypes()) so that the menu command configurers appear up with the config options that select them. So the 'Menu text to draw multiple' should appear directly below the 'Allow multiple cards to be drawn' option.

Reply:
Oh right, I think you'd had me do that "later in the PR history"

Reply:
All fixed